examples/connectivity/plot_phase_amplitude_coupling.py

```diff
@@ -0,0 +1,102 @@
+"""
+========================================================================
+Compute the modulation index for a given phase amplitude coupled signal.
+========================================================================
+
+Computes the normalized amplitudes across phase bins for a synthetic signal
+and its surrogates and display it on a phase amplitude plot. The modulation
+index is also computed and shown.
+"""
+
+# Authors: Praveen Sripad <praveen.sripad@rwth-aachen.de>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+print(__doc__)
+
+import numpy as np
+from mne.connectivity.cfc import (phase_binned_amplitude,
+                                  compute_phase_binned_stats)
+from mne.simulation.cfc import simulate_pac_signal
+from mne.viz import plot_phase_amplitude_coupling
+from matplotlib import pyplot as plt
+from scipy.stats import norm
+from mne.stats import fdr_correction
+
+# First, generate the synthetic data
+sfreq = 1000.  # sampling frequency
+
+duration, n_epochs = 1., 100  # let us create 100 trials of 1s each
+
+# set the phase modulating freq and amplitude modulated freq
+f_phase, f_amplitude = 8., 80.
+
+# To produce a realistic PAC signal, include multiple levels of modulation
+data = simulate_pac_signal(sfreq, duration, n_epochs, f_phase, f_amplitude,
+                           random_state=0, sigma=5)
+
+n_bins = 18  # number of bins
+l_phase_freq, h_phase_freq = 8., 13.  # range of phase modulating freq
+l_amp_freq, h_amp_freq = 60., 100.  # range of amplitude modulated freq
+
+# Calculate amplitude distribution and the phase bins
+amplitude_distribution, phase_bins = phase_binned_amplitude(
+    data, sfreq,
```

Comment:
This is not the same line design as in line 60.
I find it more readable if you keep the same design.